### DISCUSSION TITLE
Error on publishing

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#3177] [Spoke Questions] [answered]
I tried to publish a space that had no problem before and got this error:
<img width="406" alt="Screen Shot 2020-10-14 at 10 23 44 PM" src="https://user-images.githubusercontent.com/40791828/96080502-0f3f5e00-0e6c-11eb-97db-f6316a098ccd.png">

Does anyone know what this means? I haven't done anything different to my project since last time I published.

**Comment by hilarytsai** [comment, marked as answer]
Closing my browser and reloading it fixed this problem!

**Reply by johnshaughnessy** [reply]
I suspect what may have happened is that your browser had an older version of the code cached and when you refreshed, you fetched the latest code which included a fix for this bug. I've had this happen to me when I "reopen a closed tab" rather than re-navigating to the website.
